File: jgonzalez9.java
//Jovanny Gonzalez
//CSC 321-40

import java.util.Scanner;
import java.util.Random;
public class jgonzalez9{
	public static void main (String[]args){
	Random random= new Random();
	int y=1+random.nextInt(20);
	//int y=7;
	System.out.print("Guess a number between 1-20: ");
	Scanner keyboard=new Scanner(System.in);
	int guess=keyboard.nextInt();

	while(guess<1||guess>20){
	System.out.println("Out of Range,re-enter guess: ");
	guess=keyboard.nextInt();

	}

	if (guess!=y){
	System.out.println("Guess does not equal my random number,Y: "+y);

		if (guess>y)
		{
			System.out.println("Guess is greater than Y, you lose.");
		}
		if (guess<y)
		{
			System.out.println("Guess is less than Y, you lose.");
		}
	}

	if (guess==y)
	{
		for(int i=0;i!=y;i++){
		System.out.println("You win. Guess is equal to Y");
		}
	}

	}
}

File: jgonzalez9.c
// Jovanny Gonzalez
// CSC 321-40

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main(void)
{
	srand(time(NULL));

	int guess;
	int y=1+(rand()%21);

	while(guess<1||guess>20)
	{
		printf("Guess a number between 1-20: \n");
		scanf("%d",&guess);
	
		if(guess<1||guess>20)
		{
			printf("Out of Range,re-enter guess.\n");
		}

	}

	if (guess!=y){
	printf("Guess does not equal my random number,Y: %d\n",y);

		if (guess>y)
		{
			printf("Guess is greater than Y, you lose.\n");
		}
		if (guess<y)
		{
			printf("Guess is less than Y, you lose.\n");
		}
	}

	if (guess==y)
	{
		for(int i=0;i!=y;i++){
		printf("You win. Guess is equal to Y\n");
		}
	}



	return EXIT_SUCCESS;
}

